packages/react/src/primitives/types/style.ts

```diff
@@ -53,6 +53,13 @@ export interface BaseStyleProps extends FlexItemStyleProps, GridItemStyleProps {
   height?: ResponsiveStyle<StyleToken<Property.Height>>;
   left?: ResponsiveStyle<StyleToken<Property.Left>>;
   letterSpacing?: ResponsiveStyle<Property.LetterSpacing>;
+  lineHeight?: ResponsiveStyle<Property.LineHeight>;
+  margin?: ResponsiveStyle<Property.Margin>;
+  maxHeight?: ResponsiveStyle<Property.MaxHeight>;
+  maxWidth?: ResponsiveStyle<Property.MaxWidth>;
+  minHeight?: ResponsiveStyle<Property.MinHeight>;
+  minWidth?: ResponsiveStyle<Property.MinWidth>;
+  opacity?: ResponsiveStyle<Property.Opacity>;
```

Comment:
Could we change these lines to `ResponsiveStyle<StyleToken<Property.*>>;` ?

Reply:
Oh yea, I would be happy to do that